querydsl-sql/src/test/java/com/querydsl/sql/CoalesceTest.java

```diff
@@ -0,0 +1,21 @@
+package com.querydsl.sql;
+
+import org.junit.Test;
+
+import com.querydsl.core.types.dsl.Coalesce;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+
+public class CoalesceTest {
```

Comment:
I couldn't think where to put this... what do we think about the location of this test? Would the one individual test be better included in an existing test class somewhere?